content/en/docs/Getting started/_index.md

```diff
@@ -98,11 +148,54 @@ Specify the [Docsy theme](https://github.com/google/docsy) like any other Hugo t
     $ git submodule update --init --recursive
 
 
-You can either use the theme as a submodule like the example site, as shown above (our recommended approach for easy updates), or can just [clone the theme](https://gohugo.io/themes/installing-and-using-themes/#install-a-single-theme) into your project's `themes` subdirectory. For simplicity we recommend copying and editing our [example site configuration](#configuring-your-site) for your project or you may get Hugo errors for missing parameters and values when you try to build your site.
+You can either use the theme as a submodule like the example site, as shown above (our recommended approach for easy updates), or can just [clone the theme](https://gohugo.io/themes/installing-and-using-themes/#install-a-single-theme) into your project's `themes` subdirectory. For simplicity we recommend copying and editing our [example site configuration](#configuring-your-site) for your project, or you may get Hugo errors for missing parameters and values when you try to build your site.
+
+### Cloning the Docsy theme to your projects `themes` subdirectory
+
+    $ cd themes
+    $ git clone https://github.com/google/docsy
+
+For more information, see [Install and Use Themes](https://gohugo.io/themes/installing-and-using-themes/#install-a-single-theme) on the [Hugo](https://gohugo.io) site.
 
 ### Preview your site
 
-Whichever approach you take, we recommend running Hugo locally and taking a look at your site before you do anything else to make sure it works. You can find out how to do this in [Serving your site locally](/docs/deployment/#serving-your-site-locally). Once your site is working, you can start [adding your own content and customizations](/docs/adding-content/).
+To build and preview your site locally:
+
+    $ cd myproject
+    $ hugo server
+    
+ By default, your site will be available at http://localhost:1313/.     
+
+## Basic site configuration
+
+Site-wide configuration details and parameters are defined in your project's `config.toml` file. These include your chosen Hugo theme (Docsy, of course!), project name, community links, Google Analytics configuration, and Markdown parser parameters. See the examples with comments in [`config.toml` in the example project](https://github.com/google/docsy-example/blob/master/config.toml) for how to add this information. We recommend copying this `config.toml` and editing it even if you're just using the theme and not copying the entire Docsy example site.
+
+The Docsy example site comes with some defaults you may want to remove or customize straight away:
+
+### Internationalization
+
+By default, the Docsy example site supports Norwegian content. If you don't intend to translate your site to Norwegian, you can remove the language switcher by removing the following lines from `config.toml`:
```

Comment:
the Docsy example site supports content in English and Norwegian. You can find out more about how Docsy supports multi-language content in [Multi-language support](/docs/language/).

Reply:
done